Add 2022-12-20 agenda and minutes (#86)
solid/notifications-panel: meetings/2022-12-20.md

* SC: Let's check if that's accurate and add more clarity where needed.
* SC: Some questions/todos:
  * Clarifying the concepts/relationships between notification channel type and subscription type, e.g., https://github.com/solid/notifications-panel/blob/main/meetings/2022-12-15.md#cases-for-channels-without-subscription-needed
  * Notification Sender aside, we presume there is only one kind of a "client" and that's Subscription Client — whether that's actually going through the subscription steps or not (e.g., in the case with discovering `channel` in the Description Resource).
  * Do we need to mention "Server MUST not change `sendTo`" ? Why would a server do that and wouldn't that be out of spec behaviour?
  * https://gitter.im/solid/notifications-panel?at=63a11f64d8678973f972de3e :
    >I realized that to relate resources with their 'static' channels (which don't require subscription) we could simply use `notify:topic` in reverse direction, for JSON-LD we could just define some alias using `@reverse`

* SC: explaining terminology

> * eP: example from last week uses `service` to point to all the `SubscriptionResource`, we already have that term so I'm not sure what the `notification channel` should be.

Reviewer: ```suggestion
* eP: Example from last week uses `service` to point to all the `SubscriptionResource`.We already have that term so I'm not sure what the `notification channel` should be.
```

* SC: we could rename `service` property to `subscription`
* eP: +1 and remove `notification service` since we already have `subscription resource`
* SC: I'm going to edit it now
* RG: response is specialized by `notification channel type` not `subscription type`
* SC: Is it `subscription type` or `notification channel type`?
* RG: Last week we decided that we can have those non subscription channels
* eP: I think we should migrate from `subscription type` to `notification channel type`
* SC: should we change it now?

> * eP: I think the sooner, the better. Calling it subscription might be misleading

Reviewer: ```suggestion
* eP: I think the sooner, the better. Calling it `subscription` might be misleading
```

> * SC: There are two paths, 1st subscription path and 2nd the channel path. One can connect to existing channel without subscription step.

Reviewer: ```suggestion
* SC: There are two paths: first `subscription` path, and second the `channel` path. One can connect to existing `channel` without `subscription` step.
```

> * SC: Should we call it Channel Types or Notification Channel Types ?
> * RG: I prefer Notification Channel TYpes

Reviewer: ```suggestion
* SC: Should we call it `Channel Types` or `Notification Channel Types`?
* RG: I prefer `Notification Channel Types`
```

> ACTION: eP: create issue on considering request as requirement or preference, and can server overwrite desired channel properties or has to reject if can't satisfy

Reviewer: ```suggestion
ACTION: eP: create issue on considering request as requirement or preference, and whether server can override desired channel properties or has to reject if can't satisfy
```

```ttl
<https://channel.example/guinan/profile> notify:topic <https://example.org/guinan/profile> .
<https://example.org/guinan/profile> notify:channel <https://channel.example/guinan/profile> .

```

```jsonld=
{
    "@context": {
        "channel": { "@reverse": "notify:topic"},
        "topicOf": { "@reverse": "notify:topic"}
    }
}
```

ACTION: eP to create issue discussing need for `channel` defined in the vocab or just in JSON-LD context.

Subscription clients can use subscriptions 

> 1) to receive custom  notifications

Reviewer: ```suggestion
1) to receive custom notifications
```
